aten/src/ATen/core/ivalue.h

```diff
@@ -127,6 +127,7 @@ struct Capsule {
   _(Double)                  \
   _(ComplexDouble)           \
   _(Int)                     \
+  _(BoxedInt)                \
```

Comment:
I object to the name BoxedInt, because it implies that the integers are "boxed", which they are not; they fit in 64-bit. I'm not sure if we should have the same naming convention in IValue as in JIT schema, something that needs to be thought about...